tests/components/plugwise/test_init.py

```diff
@@ -53,5 +56,40 @@ async def test_unload_entry(hass, mock_smile_adam):
     entry = await async_init_integration(hass, mock_smile_adam)
 
     mock_smile_adam.async_reset = AsyncMock(return_value=True)
-    assert await async_unload_entry(hass, entry)
+    await hass.config_entries.async_unload(entry.entry_id)
+    await hass.async_block_till_done()
+    assert entry.state == ENTRY_STATE_NOT_LOADED
     assert not hass.data[DOMAIN]
+
+
+async def test_unload_gateway_entry(hass, mock_smile_adam):
+    """Test being able to unload an entry."""
+    entry = await async_init_integration(hass, mock_smile_adam)
+
+    mock_smile_adam.async_reset = AsyncMock(return_value=True)
```

Comment:
im not sure we need to test unloading twice?

Reply:
This was partially there for `gateway` as well as upcoming `usb` but you are right, removing